tests/validate/bandwidth_test/src/host.cpp

```diff
@@ -247,6 +259,18 @@ int main(int argc, char** argv) {
                 }
             }
 
+            for (int i = 0; i < num_kernel_ddr; i++) {
+               cl_ulong start_time = krnl_events[i].getProfilingInfo<CL_PROFILING_COMMAND_START>();
+               cl_ulong end_time = krnl_events[i].getProfilingInfo<CL_PROFILING_COMMAND_END>();
+               double dnsduration = (double)(end_time - start_time) / reps; //nanoseconds to seconds
+               double dsduration = dnsduration / ((double)1000000000); // Convert the duration from nanoseconds to seconds
+               double bpersec = (data_size) / dsduration;
+               double mbpersec = (2 * bpersec) / ((double)1024 * 1024); // Convert b/sec to mb/sec
```

Comment:
Overall , change is looking good, Doing similar calculations below. Please create a common function .

Reply:
Modified the code according to suggestion. Thank you Vamshi.